Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
To be honest if this works now, which it seems to, this is an appreciably cleaner way of achieving what we were already achieving. Less Homebrew-specific hacks, simpler paths, and less code to maintain indefinitely.

If me, you & Christian all got smacked by a truck tomorrow I feel like this would be less obtuse for whichever poor soul took the job over.